src/install.rs

```diff
@@ -29,7 +29,7 @@ pub async fn execute_transaction(
     let package_cache = PackageCache::new(cache_dir.join("pkgs"));
 
     // Create an install driver which helps limit the number of concurrent filesystem operations
-    let install_driver = InstallDriver::default();
+    let install_driver = InstallDriver::new(100, Some(prefix_records));
```

Comment:
Was this  100 a guess or is it based on something?

Reply:
It's the value we are currently using as default (so it's the same as before).